.github/workflows/pr-code-format.yml

```diff
@@ -72,9 +58,17 @@ jobs:
         # explicitly in code-format-helper.py and not have to diff starting at
         # the merge base.
         run: |
-          python ./code-format-tools/llvm/utils/git/code-format-helper.py \
+          python ./llvm/utils/git/code-format-helper.py \
+            --write-comment-to-file \
             --token ${{ secrets.GITHUB_TOKEN }} \
             --issue-number $GITHUB_PR_NUMBER \
             --start-rev $(git merge-base $START_REV $END_REV) \
```

Comment:
The start revision and end revision I think would need to be adjusted? Now that this is a `pull_request` workflow, we get a merge commit merging the PR into `main` (or whatever the target branch is), so I think we can probably just do `HEAD~1` and `HEAD`? I can't remember what Github ends up setting `base.sha` and `head.sha` to here.

Reply:
Yeah I am a bit worried about this. It really needs some testing, it was quite difficult to get the right incantation that formats just what have changed, but as long as we test that we format the right things it should  be fine.

Reply:
What are the different scenarios we need to test?

Reply:
Actually, we should be fine here as long as we're checking out the same ref that we were before. As long as we're doing that, everything should work the same. It will have some quirks like not being run if there's a merge conflict even if we're not performing a merge, but that should be fine.

Eventually if we want to clean this up, it's definitely doable, but keeping the current functionality (minus running when there are merge conflicts) as it is seems fine to me.